docs/content/guides/getting-started/demo.md

```diff
@@ -12,21 +12,36 @@ tags:
 
 [[toc]]
 
+::: only-for javascript
 Play around with a demo of Handsontable, in your favorite framework.
+:::
+
+::: only-for react
+Play around with a demo of Handsontable, in React.
+:::
+
+<div class="example-container">
+  <div id="example"></div>
+  <script defer src="https://examples.handsontable.com/examples/12.1.1/docs/js/demo/src.3f3ef8b8.js"></script>
+  <style>
+    @import 'https://examples.handsontable.com/examples/12.1.1/docs/js/demo/src.4d67a99b.css';
+  </style>
+</div>
 
-<div class="example-container"><iframe
-    src="https://handsontable.github.io/handsontable/examples/next/docs/js/demo/"
-    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone;
-      midi; payment; usb; vr; xr-spatial-tracking"
-    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
-  ></iframe></div>
 ## Find the code on GitHub
 
+::: only-for javascript
 - [JavaScript demo app](https://github.com/handsontable/handsontable/tree/develop/examples/next/docs/js/demo/)
 - [TypeScript demo app](https://github.com/handsontable/handsontable/tree/develop/examples/next/docs/ts/demo/)
-- [Angular demo app](https://github.com/handsontable/handsontable/tree/develop/examples/next/docs/angular/demo/)
 - [React demo app](https://github.com/handsontable/handsontable/tree/develop/examples/next/docs/react/demo/)
+- [Angular demo app](https://github.com/handsontable/handsontable/tree/develop/examples/next/docs/angular/demo/)
 - [Vue demo app](https://github.com/handsontable/handsontable/tree/develop/examples/next/docs/vue/demo/)
+:::
+
+::: only-for react
```

Comment:
I don't understand why we hide other examples when the documentation user chooses to read it in React version.